toffy/settings.py

```diff
@@ -9,11 +9,29 @@
 COREG_SAVE_PATH = 'C:\\Users\\Customer.ION\\Documents\\toffy\\toffy\\coreg_params.json'
 FIDUCIAL_POSITIONS = ['top left', 'top right', 'middle left', 'middle right',
                       'bottom left', 'bottom right']
+
 MICRON_TO_STAGE_X_MULTIPLIER = 0.001001
 MICRON_TO_STAGE_X_OFFSET = 0.3116
 MICRON_TO_STAGE_Y_MULTIPLIER = 0.001018
 MICRON_TO_STAGE_Y_OFFSET = 0.6294
 
+STAGE_LEFT_BOUNDARY = -1.03
```

Comment:
How were these values determined? This was what you got on Creed or Drago?

Reply:
@ngreenwald yeah I got these from Creed. Should be pretty similar to Drago because the co-registration coefficients on each were about the same.

Reply:
Got it. Since the boundaries are checked directly, and don't have any padding added, I think we should be a bit conservative and expand them a bit, similar to the tolerance we add for the other parameters

Reply:
Ok sounds good. A `tol` multiplier of 0.2 is probably not going to work as well for slide boundaries (especially for optical) because the distances between top-bottom and left-right are significantly larger.

We can either use a smaller `tol` multiplier or a fixed `tol` distance (set pretty small, ex. 100 optical pixels or 1 stage coordinate), also open to other ideas.

Reply:
Yeah, I think adding/subtracting a constant is a good fix for now. We can revisit if this becomes an issue